<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="PageAdmin">
     <widget class="QPushButton" name="ButtonSearchBA">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>书籍查询</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonBuyA">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>购书</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidgetA">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>921</width>
        <height>681</height>
       </rect>
      </property>
      <widget class="QWidget" name="PageSearchBA"/>
      <widget class="QWidget" name="pageSearchCA">
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>881</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QLabel" name="labelSchoolS">
           <property name="text">
            <string>学院：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="comboBoxType"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineName"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelNoS">
           <property name="text">
            <string>账号：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelTypeS">
           <property name="text">
            <string>类型：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="comboBoxSchool"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditNo"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelNameS">
           <property name="text">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="ButtonClientSearch">
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelId">
           <property name="text">
            <string>学号/工作号：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineCId"/>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="tableClient">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>881</width>
          <height>401</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>姓名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>编号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>类型</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>学院</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="PageBuyA">
       <widget class="QWidget" name="gridLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>761</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelBuyName">
           <property name="text">
            <string>书名：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineBuyName"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelBuyNumber">
           <property name="text">
            <string>数量：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineBuyAmount"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelBuyCategory">
           <property name="text">
            <string>类型：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labelBuyPrice">
           <property name="text">
            <string>价格：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="lineBuyPrice"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="BoxBuyCategory"/>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineBuyAuthor"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="BoxBuyPress"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelBuyPress">
           <property name="text">
            <string>出版社：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelBuyAuthor">
           <property name="text">
            <string>作者：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButtonBuyConfirm">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>420</y>
          <width>93</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>确认购入</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelBuyHint">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>420</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="PageHelpA"/>
     </widget>
     <widget class="QPushButton" name="ButtonSearchCA">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>用户查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>111</width>
        <height>701</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>ButtonBuyA</zorder>
     <zorder>ButtonSearchBA</zorder>
     <zorder>ButtonSearchCA</zorder>
     <zorder>stackedWidgetA</zorder>
    </widget>
    <widget class="QWidget" name="PageClient">
     <widget class="QPushButton" name="ButtonSearch">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>查询/借书</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonReturn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>还书</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>个人信息</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidgetC">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>921</width>
        <height>691</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="PageSearch">
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>871</width>
          <height>191</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="LabelBName">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>书名：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="LineBookNumber">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="LabelBNumber">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>编号：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="LabelPress">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>出版社：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="LabelCategory">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>类型</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="LabelAuthor">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>作者：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="CBoxType">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="LineBookWriter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="CBoxPress">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LineBookName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="ButtonConfirm">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="tableResult">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>881</width>
          <height>431</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>书名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>作者</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>出版社</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>类型</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>剩余数量</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="TextResult">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>结果：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ButtonRate">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>210</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>借阅率</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="PageReturn">
       <widget class="QTableWidget" name="tableReturn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>911</width>
          <height>491</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>书名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>编号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>借书时间</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>剩余时间</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="PageInfo">
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>861</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="LayoutInfo">
         <item row="1" column="2">
          <widget class="QLabel" name="labelNumber">
           <property name="text">
            <string>账号：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelCredit">
           <property name="text">
            <string>信用：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="TextSchool">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelSchool">
           <property name="text">
            <string>学院：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="TextPw">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="TextCName">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="TextCredit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelType">
           <property name="text">
            <string>类型：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="TextType">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelNumberA">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labelPw">
           <property name="text">
            <string>密码：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelID">
           <property name="text">
            <string>学号/工作号：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="TextID">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="labelInfo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>72</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>基本信息：</string>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabBook">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>871</width>
          <height>361</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabBorrowed">
         <attribute name="title">
          <string>已借阅</string>
         </attribute>
         <widget class="QTableWidget" name="tableBorrowed">
          <property name="geometry">
           <rect>
            <x>-5</x>
            <y>1</y>
            <width>871</width>
            <height>331</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>书名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>编号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>借书时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>剩余天数</string>
           </property>
          </column>
         </widget>
        </widget>
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string>已预约</string>
         </attribute>
         <widget class="QTableWidget" name="tableOrdered">
          <property name="geometry">
           <rect>
            <x>-5</x>
            <y>1</y>
            <width>871</width>
            <height>321</height>
           </rect>
          </property>
          <column>
           <property name="text">
            <string>书名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>预约时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>是否有余</string>
           </property>
          </column>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="PageHelp"/>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-9</x>
        <y>0</y>
        <width>101</width>
        <height>691</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>ButtonSearch</zorder>
     <zorder>ButtonReturn</zorder>
     <zorder>ButtonInfo</zorder>
     <zorder>stackedWidgetC</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
